esp/src/src-react/components/forms/landing-zone/FileListForm.tsx

```diff
@@ -61,6 +66,8 @@ export const FileListForm: React.FunctionComponent<FileListFormProps> = ({
 
     const closeForm = React.useCallback(() => {
         setShowForm(false);
+        const uploaderBtn = document.querySelector("#uploaderBtn");
+        uploaderBtn["value"] = null;
```

Comment:
Is this the only way?

Reply:
It seemed like the most straightforward way. I believe the issue was Fluent didn't have a Button that would function like an input type="file", so there's just a hidden element on the page that is triggered via the onClick of the "Upload" button on the page (https://github.com/hpcc-systems/HPCC-Platform/blob/master/esp/src/src-react/components/LandingZone.tsx#L366).